src/core/layout/qgslayoutitemelevationprofile.h

```diff
@@ -221,6 +242,8 @@ class CORE_EXPORT QgsLayoutItemElevationProfile: public QgsLayoutItem
 
     double mTolerance = 0;
 
+    std::unique_ptr<QgsLineSymbol> mSubsectionsSymbol;
```

Comment:
I can't see where you are saving/restoring this symbol -- am I missing something?

Reply:
I had forgotten to implement it. This is fixed now.